test/distributed/org/apache/cassandra/distributed/test/NativeProtocolTest.java

```diff
@@ -18,58 +18,61 @@
+                                              .withConfig(config -> config.with(GOSSIP, NETWORK, NATIVE_PROTOCOL))
+                                              .start()))
         {
-            final com.datastax.driver.core.Cluster cluster = com.datastax.driver.core.Cluster.builder().addContactPoint("127.0.0.1").build();
-            Session session = cluster.connect();
-            session.execute("CREATE TABLE " + KEYSPACE + ".tbl (pk int, ck int, v int, PRIMARY KEY (pk, ck));");
-            session.execute("INSERT INTO " + KEYSPACE + ".tbl (pk, ck, v) values (1,1,1);");
-            Statement select = new SimpleStatement("select * from " + KEYSPACE + ".tbl;").setConsistencyLevel(ConsistencyLevel.ALL);
-            final ResultSet resultSet = session.execute(select);
-            assertRows(resultSet, row(1, 1, 1));
-            Assert.assertEquals(3, cluster.getMetadata().getAllHosts().size());
-            session.close();
-            cluster.close();
+
+            try (com.datastax.driver.core.Cluster cluster = com.datastax.driver.core.Cluster.builder().addContactPoint("127.0.0.1").build();
+                 Session session = cluster.connect())
+            {
+                session.execute("CREATE TABLE " + KEYSPACE + ".tbl (pk int, ck int, v int, PRIMARY KEY (pk, ck));");
+                session.execute("INSERT INTO " + KEYSPACE + ".tbl (pk, ck, v) values (1,1,1);");
+                Statement select = new SimpleStatement("select * from " + KEYSPACE + ".tbl;").setConsistencyLevel(ConsistencyLevel.ALL);
+                final ResultSet resultSet = session.execute(select);
+                assertRows(RowUtil.toObjects(resultSet), row(1, 1, 1));
+                Assert.assertEquals(3, cluster.getMetadata().getAllHosts().size());
+            }
         }
     }
 
     @Test
     public void withCounters() throws Throwable
     {
-        try (Cluster dtCluster = init(Cluster.create(3,
-                config -> config.with(GOSSIP, NETWORK, NATIVE_PROTOCOL))))
+        try (ICluster ignored = init(builder().withNodes(3)
+                                              .withConfig(config -> config.with(GOSSIP, NETWORK, NATIVE_PROTOCOL))
+                                              .start()))
         {
             final com.datastax.driver.core.Cluster cluster = com.datastax.driver.core.Cluster.builder().addContactPoint("127.0.0.1").build();
             Session session = cluster.connect();
             session.execute("CREATE TABLE " + KEYSPACE + ".tbl (pk int, ck counter, PRIMARY KEY (pk));");
             session.execute("UPDATE " + KEYSPACE + ".tbl set ck = ck + 10 where pk = 1;");
             Statement select = new SimpleStatement("select * from " + KEYSPACE + ".tbl;").setConsistencyLevel(ConsistencyLevel.ALL);
             final ResultSet resultSet = session.execute(select);
-            assertRows(resultSet, row(1, 10L));
+            assertRows(RowUtil.toObjects(resultSet), row(1, 10L));
```

Comment:
In short, for some reason static imports didn't work well with a static method in the parents class and compiler couldn't figure out which `assertRows` I was referring to. And I couldn't add `assertRows(ResultSet)` to the parent class because it has a driver dependency.